Composer/packages/client/src/components/WebChat/hooks/useTranscriptFocusListener.ts

```diff
@@ -33,12 +34,15 @@ export const useTranscriptFocusListener = () => {
         }
       });
       if (trafficItem && currentProjectId) {
-        setDebugPanelActiveTab(WebChatInspectorTabKey);
+        // only switch to the Web Chat tab if no tab is active
```

Comment:
comment is about the logic above this line: if activity (first arg) doesn't have a value, maybe return right away?

Reply:
Are you talking about just adding:

```javascript
useCallback(
  ({ activity }: { activity: Activity }) => {
    if (!activity) {
      return;
    }
```

Reply:
yep, what do you think?

Reply:
I don't think the [hook ever gets called with an `undefined` value](https://github.com/microsoft/BotFramework-WebChat/blob/3fc2853cc53bd4cf49950862ede25af7f923e268/packages/component/src/hooks/useObserveTranscriptFocus.ts#L7), but it doesn't hurt

Reply:
try adding a hero card with actions and click those action buttons, activity seems ```undefined``` at that point

Reply:
let me give that a shot

Reply:
You're right! I'll make the change